src/pages/PlanPage/subRoutes/CurrentOrgPlan/BillingDetails/Address/AddressForm.tsx

```diff
@@ -22,15 +23,49 @@ function AddressForm({
   owner,
 }: AddressFormProps) {
   const elements = useElements()
+  useEffect(() => {
+    const updateAppearance = () => {
+      const isDarkMode = document.body.classList.contains('dark')
+
+      elements?.update({
+        appearance: {
+          variables: {
+            fontFamily: 'Poppins, ui-sans-serif, system-ui, sans-serif',
+          },
+          rules: {
+            '.Label': {
+              fontWeight: '600',
+              color: isDarkMode ? 'rgb(210,212,215)' : 'rgb(14,27,41)', // Same values as --color-app-text-primary.
+            },
+            '.Input': {
+              backgroundColor: isDarkMode
+                ? 'rgb(22,24,29)'
+                : 'rgb(255,255,255)', // Same values as --color-app-container.
+              borderColor: isDarkMode ? 'rgb(47,51,60)' : 'rgb(216,220,226)', // Same values as --color-ds-gray-tertiary.
+              color: isDarkMode ? 'rgb(210,212,215)' : 'rgb(14,27,41)', // Same values as --color-app-text-primary.
+            },
```

Comment:
Are you able to just reference `var(--<var name>)` here instead of setting direct values?

Reply:
Unfortunately not for these, stripe can't reference style sheet on component mount

Reply:
that's dumb

Reply:
Added!